ci/uninstall-thin-edge_io/uninstall-thin-edge_io.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/sh
+
+set -e
+
+usage() {
+    cat <<EOF
+USAGE:
+    delete-thin-edge_io [COMMAND]
+    
+COMMANDS:
+    remove     Uninstall thin-edge.io with keeping configuration files
+    purge      Uninstall thin-edge.io and also remove configuration files  
+
+EOF
+}
+
+stop_a_service_if_running() {
+    local status
+    status=$(systemctl show -p ActiveState --value "$1")
+    if [ "$status" = "active" ]; then
+        sudo systemctl stop "$1"
+    fi
+}
+
+stop_services() {
```

Comment:
```suggestion
stop_extension_services() {
```
Just to be clearer as I was wondering why you were not stopping the tedge-agent or mapper services before seeing the rest of the code.

Reply:
Renamed it.